src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -217,8 +217,8 @@
               <!-- Warning if orgPerson has no roles -->
               <div v-if="orgPerson.roles.length > 0">
                 <v-col
-                  v-for="(role, index) in orgPerson.roles"
-                  :key="index"
```

Comment:
eslint didn't like "index" here because we're already in a loop with an index variable, so I renamed this one.